app/javascript/components/visualization/controls/ScatterPlotLegend.js

```diff
@@ -94,7 +94,10 @@ function LegendEntry({
       onClick={() => toggleSelection()}
     >
       <div className="scatter-legend-icon" style={iconStyle}></div>
-      <div className="scatter-legend-entry">{entry}</div>
+      <div className="scatter-legend-entry">
+        <span className="legend-label" title={entry}>{entry}</span>
+        <span className="num-points" title={`${numPoints} points have this label`}>{numPoints}</span>
```

Comment:
I wonder if 'X points are in the {entry} group'  or just 'X points are in this group' might be a clearer title.  Probably a good demo mini-discussion (the former might run into length issues)

Reply:
Good suggestions.  Per demo feedback, I went with "`${numPoints} points in this group`" (8855a28).